althea_kernel_interface/src/manipulate_uci.rs

```diff
@@ -28,7 +28,7 @@ impl KernelInterface {
 
     //Sets an arbitrary UCI list on OpenWRT
     pub fn set_uci_list(&self, key: &str, value: &[&str]) -> Result<bool, Error> {
-        self.del_uci_var(&key);
+        self.del_uci_var(&key)?;
```

Comment:
list might not exist, so it is not necessary to bail here

Reply:
I'd log with `trace!` here